scripts/cromwell_validate.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Source the utility script with the find_tool function
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+source "$SCRIPT_DIR/find_tool.sh"
```

Comment:
This isn't working locally for me:

```
skross wdl-unit-tests % make test_java_validate 
Validating WDL files with womtool validate...
scripts/cromwell_validate.sh: line 8: find_tool: command not found
```

It needs:

```suggestion
source "$SCRIPT_DIR/scripts/find_tool.sh"
```

or something.

Reply:
@seankross forgot to bump you last comment